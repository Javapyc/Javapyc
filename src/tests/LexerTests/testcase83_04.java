while asdf764JKL; extends void && main.{boolean >= false] return simple0class 
if !static System.out.println(/* commenting stuff here ...
*/// this should stay commented