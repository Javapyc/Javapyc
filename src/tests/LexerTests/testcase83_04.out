ReservedWord, while
ID, asdf764JKL
Delimiter, ;
ReservedWord, extends
ReservedWord, void
Operator, &&
ReservedWord, main
Delimiter, .
Delimiter, {
ReservedWord, boolean
Operator, >=
ReservedWord, false
Delimiter, ]
ReservedWord, return
ID, simple0class
ReservedWord, if
Operator, !
ReservedWord, static
ReservedWord, System.out.println
Delimiter, (
